packages/mui-material/package.json

```
@@ -60,10 +60,10 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.14.8",
-    "@mui/core": "5.0.0-alpha.46",
-    "@mui/system": "5.0.0-rc.1",
-    "@mui/types": "7.0.0-rc.1",
-    "@mui/utils": "5.0.0-rc.1",
+    "@mui/core": "5.0.0-alpha.47",
+    "@mui/system": "5.0.0",
```

These should use caret ranges 
```suggestion
    "@mui/system": "^5.0.0",
```

This applies to all dependencies on stable releases i.e. alpha release should stay pinned.

---

Done 👍